paywall/src/unlock.js/startup.ts

```diff
@@ -115,11 +115,13 @@ export function startup(
   // go!
   mainWindow.init()
   if (launchModal) {
-    iframes.data.once(PostMessages.UPDATE_ACCOUNT, accountAddress => {
+    const listener = (accountAddress: string | null) => {
```

Comment:
`listener` will be called every time we get a `PostMessages.UPDATTE_ACCOUNT`, until the message carries an account address, at which point the listener will be cleared.